### PR TITLE
Remote package docs

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: docs/content/en/docs/reference/clusterspec/packages.md [hunk 1]
```diff
@@ -32,16 +29,16 @@ metadata:
...
-    source:
-      registry: public.ecr.aws/eks-anywhere
-      repository: hello-eks-anywhere
-      versions:
-      - digest: sha256:da25f5fdff88c259bb2ce7c0f1e9edddaf102dc4fb9cf5159ad6b902b5194e66
-        name: v0.0
-        images:
-        - digest: sha256:da25f5fdff88c259bb2ce7c0f1e9edddaf102dc4fb9cf5159ad6b902b5194e66
-          repository: public.ecr.aws/eks-anywhere
+    - name: hello-eks-anywhere
```
Comment (ivyostosh) [inline docs/content/en/docs/reference/clusterspec/packages.md:32]: Is the indentation here correct?

Comment (TerryHowe) [reply docs/content/en/docs/reference/clusterspec/packages.md:32]: https://github.com/aws/eks-anywhere/blob/82174e425244738eb8e1ea1e3b072306d4490ca0/docs/content/en/docs/reference/clusterspec/packages.md 

Looks correct, I pasted that from actual

**Pull request merged**